platform-sdk/swirlds-platform-core/src/testFixtures/java/com/swirlds/platform/test/fixtures/crypto/PreGeneratedX509Certs.java

```diff
@@ -175,8 +81,8 @@ private static void loadCerts() {
         final InputStream sigCertIs;
         final InputStream agreeCertIs;
         try {
-            sigCertIs = ResourceLoader.loadFileAsStream(SIG_CERT_FILE);
-            agreeCertIs = ResourceLoader.loadFileAsStream(AGREE_CERT_FILE);
+            sigCertIs = PreGeneratedX509Certs.class.getResourceAsStream(SIG_CERT_FILE);
+            agreeCertIs = PreGeneratedX509Certs.class.getResourceAsStream(AGREE_CERT_FILE);
```

Comment:
In the modules system, _resources_ need to follow the same restrictions as Java classes. If the resources are local to a module, they need to be loaded from inside the Module, which we can do via `Class.getResourceAsStream` where the referenced class is in the same Module as the resource to load.

Hence the `ResourceLoader` test utility class in it's current form is not very useful. It was not used in many places and there is also unused code in there. I removed the unused (and now not useful anymore) code from that class as part of this PR.